library/core/src/unicode/mod.rs

```diff
@@ -1,6 +1,19 @@
 #![unstable(feature = "unicode_internals", issue = "none")]
 #![allow(missing_docs)]
 
+// The `pub use` ones are for use in alloc, and are not re-exported in std.
+
+pub(crate) use unicode_data::alphabetic::lookup as Alphabetic;
+pub use unicode_data::case_ignorable::lookup as Case_Ignorable;
+pub use unicode_data::cased::lookup as Cased;
+pub(crate) use unicode_data::cc::lookup as Cc;
+pub use unicode_data::conversions;
+pub(crate) use unicode_data::grapheme_extend::lookup as Grapheme_Extend;
+pub(crate) use unicode_data::lowercase::lookup as Lowercase;
+pub(crate) use unicode_data::n::lookup as N;
+pub(crate) use unicode_data::uppercase::lookup as Uppercase;
+pub(crate) use unicode_data::white_space::lookup as White_Space;
```

Comment:
Hm. Is it not possible to separate these between the `pub(crate)` and `pub use` set?

Reply:
No, rustfmt doesn't consider the visibility when grouping and ordering.

Reply:
Ah, it is a requested feature, it seems: https://github.com/rust-lang/rustfmt/issues/5083#issuecomment-1165556744

Reply:
Update: I discovered you can use `#[rustfmt::skip]` to create arbitrary custom groups, which allows separation of `pub use` from `use` if desired. See #127950 for an update to this exact piece of code.